.github/workflows/auto_assignment.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Auto Assign'
+on:
+  pull_request_target:
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+
+jobs:
+  add-assignees: # This needed for to create the gh issue in case of failed auto0-cherry-pick
+    name: Add author to assignee
+    if: "!contains(github.event.pull_request.labels.*.name, 'Auto_Cherry_Picked')"
```

Comment:
I think, dependabot PRs shouldn't be auto-assigned as well, so could you include the `dependencies` label here?

Reply:
dependabot is not a valid assignee so, anyways it will not be get assigned.